t/model/search.t

```diff
@@ -59,6 +59,17 @@ ok( $search->_not_rogue, '_not_rogue' );
     is( @{ $results->{results} }, 5, '5 results on author search' );
 }
 
+{
+    my $results = $search->search_web('author:Mo BadPod');
+    isnt( @{ $results->{results} },
+        0, '>0 results on author search with extra' );
+}
+
+{
+    eval { $search->search_web('usr/bin/env') };
```

Comment:
We also could have used `Test::Fatal` here. Not sure if that's much of a problem, though.

Reply:
I was aiming for no extra deps ;-)

Reply:
:) It must be a secondary dep, since we do already use it: https://github.com/metacpan/metacpan-api/blob/master/t/api/controller/admin.t#L5

Reply:
Although, look at that test, I'm not sure we're actually making use of it there.

Reply:
It wasn't - zapped in another commit at your request :-)